.github/CODEOWNERS

```diff
@@ -35,7 +35,7 @@
 # ServiceOwner: @Pilchie
 # ServiceLabel: %Cosmos
 # PRLabel: %Cosmos
-/sdk/cosmos/          @analogrelay @Pilchie
+/sdk/cosmos/          @analogrelay @Pilchie @kirankumarkolli @tvaron3 @FabianMeiswinkel @kundadebdatta
```

Comment:
For aliases to be valid in our `CODEOWNERS` file they need to have the following: 
- They need to be **PUBLIC** members of the Azure and microsoft GitHub Orgs
- They need to have write permission to the Azure SDK repos

❌@analogrelay, @kirankumarkolli and @FabianMeiswinkel are missing **public** membership to the microsoft Org.
❌@analogrelay and @tvaron3 are missing **public** membership to the Azure Org

✅@kundadebdatta and @Pilchie have all necessary permissions. 

Please view our [onboarding docs for getting access](https://aka.ms/azsdk/access) and go through the steps in the `Request access to Azure REST and SDK repositories` section. 

You can use the [`Validate-AzsdkCodeOwner.ps1`](https://github.com/Azure/azure-sdk-tools/blob/main/tools/github/scripts/Validate-AzsdkCodeOwner.ps1) script from our tools repo to test if your alias has all the required permissions. 
Ex: 
![image](https://github.com/user-attachments/assets/17e53c03-7471-4e2d-a592-ad82e1a5dafc)
![image](https://github.com/user-attachments/assets/99cc8cd9-2661-4538-81da-d859ddb0ccf7)

Reply:
I did make my profile public fir both MS and Azure. 
I don't have access right to the tools repository.

Reply:
I've updated my memberships to be public and am now showing as valid in the script✅. @kirankumarkolli I also see you are now valid, according to the script.

![image](https://github.com/user-attachments/assets/ca96ab55-c090-47ab-a054-9cd4ea46b69f)

Reply:
Thank you @analogrelay!

Reply:
FYR Fabian is OOF right now.

Reply:
> I did make my profile public fir both MS and Azure. I don't have access right to the tools repository.

According to the script, your membership to the `microsoft` GitHub Org isn't public: 
![image](https://github.com/user-attachments/assets/49713aaf-109d-4406-a1bc-b3ec41f6ebf6)

Reply:
@ronniegeraghty I see Kiran as valid now (see above, and I just ran the script again to confirm the same result). Are you still seeing that it's showing as invalid?

@kirankumarkolli I think we can wait for Fabian to return, so he can fix his membership visibility, before merging this. The PR volume in the Cosmos SDK is pretty low (it's basically just me right now 😅 ) and I can manually add reviewers to my PRs for now.

Reply:
> @ronniegeraghty I see Kiran as valid now (see above, and I just ran the script again to confirm the same result). Are you still seeing that it's showing as invalid?

I can confirm that Kiran's permission are showing up correctly now: 
![image](https://github.com/user-attachments/assets/fd8dfea2-d148-4742-bd58-8f89523dd960)

Reply:
If all is good, can you approve the PR and merge?